sections/oxygen_mission_execution.md

```diff
@@ -79,16 +75,18 @@ The combined data collected following these (CTD and lab calibration) steps will
 
 *NOTE: Save all data from the profiles and calibrations before deploying the glider. As noted in {refnum}`sensor_configuration` Always record the phase readings {cite}`Coppola2013`.*
 
-<!--
-![GEOMAR oxygen data loggers (Aanderaa Optode mounted on data logger) attached to a CTD frame and prepared for in-situ calibration during a CTD cast. Panels in the upper row show fixation with straps and zip ties. Panel in the lower row shows fixation with scaffolding clamps and tape in the interior lower part of the CTD frame.](/images/Datalogger.png)
--->
-
 :::{figure-md} datalogger
 <img src="../images/Datalogger.png" alt="Data logger on CTD" class="bg-primary mb-1" width="600px">
 
 GEOMAR oxygen data loggers (Aanderaa Optode mounted on data logger) attached to a CTD frame and prepared for in-situ calibration during a CTD cast. Panels in the upper row show fixation with straps and zip ties. Panel in the lower row shows fixation with scaffolding clamps and tape in the interior lower part of the CTD frame.
 :::
 
+:::{figure-md} CTD_calibration_stop
+<img src="../images/CTD_calibration_01_03.png" alt="CTD calibration stop full cast" class="bg-primary mb-1" width="600px">
```

Comment:
Was this figure copied from Johannes's thesis or created for the SOP? There is a a black shaded region which doesn't look great.

Reply:
@hahn-johannes prepared the figure for the SOP. So no copy right issues here. 

I will try to do better in combining both png's. If anyone has good tools let me know.

Reply:
I uploaded a new merged version.